deployments/helm-chart/templates/controller-daemonset.yaml

```diff
@@ -191,10 +191,10 @@ spec:
           - -external-service={{ .Values.controller.reportIngressStatus.externalService }}
 {{- else if and (.Values.controller.service.create) (eq .Values.controller.service.type "LoadBalancer") }}
           - -external-service={{ include "nginx-ingress.serviceName" . }}
+{{- end }}
 {{- end }}
           - -enable-leader-election={{ .Values.controller.reportIngressStatus.enableLeaderElection }}
           - -leader-election-lock-name={{ include "nginx-ingress.leaderElectionName" . }}
```

Comment:
This should be under an if statement here as well - this should only be set if `enableLeaderElection` is set (not a blocker, but neater).